src/App.vue

```diff
@@ -21,17 +20,10 @@ const i18n = useI18n()
 const dispatcher = { dispatcher: 'App.vue' }
 
 let debouncedOnResize
-const error = computed(() => {
-    if (store.state.ui.errors.size > 0) {
-        return store.state.ui.errors.values().next().value
-    }
-    return null
-})
-const warning = computed(() => {
-    if (store.state.ui.warnings.size > 0) {
-        return store.state.ui.warnings.values().next().value
-    }
-    return null
+const showFeedbackPopup = computed(() => {
+    console.error(store.state.ui.errors)
+    console.error(store.state.ui.warnings)
```

Comment:
don't use `console.error` but `log.error` instead (if this makes sense to stay long term in the code, and isn't some debugging help for your PR)

Reply:
good catch, I'm fairly sure this is a debugging help :)